examples/llama7b_w4a16_quantization.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<p>I think it would be nice to have a chance to validate those notebooks locally, like I do in <code>compressed-tensors</code> . This would mean setting up a flow that lets you run notebooks and check whether they terminate without an error.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuralmagic/sparseml/pull/2285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Yeah thats a good idea, though we probably don't want to run these 7b models as part of our regular testing. Maybe we could set them up to run weekly?